### ISSUE TITLE
I wish /docs/user-guide/kubectl/kubectl/kubectl_get/ existed.

### ISSUE THREAD

**Issue by rwillmer** [kubernetes/kubernetes.github.io#222]
I wish /docs/user-guide/kubectl/kubectl/kubectl_get/ existed.

**Comment by johndmulhausen** [comment]
@rwillmer Where is there a link to this URL? Can't find any amongst the links I see for kubectl_get in the repo

**Comment by bgrant0607** [comment]
This exists:
https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/kubectl/kubectl_get.md

**Comment by bgrant0607** [comment]
And this works for me:
http://kubernetes.io/docs/user-guide/kubectl/kubectl_get/

**Comment by johndmulhausen** [comment]
@bgrant0607 Right, but he's saying he's looking for /kubectl/kubectl/kubectl_get -- and that's just too many dang kubectls.

**Comment by bgrant0607** [comment]
Maybe a relative link with no leading slash somewhere?

**Comment by rwillmer** [comment]
Go to http://kubernetes.io/docs/user-guide/kubectl/kubectl_get/.

Scroll down to end of page; "See Also". Click on 'kubectl'. Error 100% reproducible

**Comment by bgrant0607** [comment]
Yep, relative links in the auto-generated pages:
```
* [kubectl](kubectl/)	 - kubectl controls the Kubernetes cluster manager
```

cc @janetkuo @pwittrock

**Comment by janetkuo** [comment]
Go to http://kubernetes.io/docs/user-guide/kubectl/kubectl/ and all the links to kubectl commands are broken.

**Comment by johndmulhausen** [comment]
Fixed in https://github.com/kubernetes/kubernetes.github.io/commit/5789c874285dd261a5d3afc7333a55900de41df8

**Comment by johndmulhausen** [comment]
The deal with this is my import script for these pages needs to do some scrubbing so we have a permanent fix. I just made these fixes by hand. GitHub Pages' "pretty" URL system adds a slash to the end, thus making relative links break, which is one reason why we had to go "absolute URL" 

Should be a temporary state of affairs anyway; @lavalamp and I have talked about generating refdocs based on a YAML representation of the APIs and CLIs that will make this import-and-munge system obsolete.
